EDU-3244-cache-settings-cli-pt (#443)
aziontech/docs: src/content/docs/pt-br/pages/devtools/cli/azion-cli-alpha/cache-settings/cache-settings.mdx

---
title: Cache settings
description: Crie, liste, descreva, atualize e exclua suas configurações de cache na plataforma da Azion através da Azion CLI.
meta_tags: cli, cache, devops
namespace: documentation_cli_cache_settings
menu_namespace: cliMenuAlpha
permalink: /documentacao/produtos/cli/cache-settings-crud/
---

## Criar

### Uso

```bash
$ azion create cache-setting --application-id 1673635839 --name "phototypesetting"
```

### Flags obrigatórias

#### application-id

A flag `--application-id` define o identificador único para a edge application.

#### name

> A flag `--name` define o nome da configuração de Cache Settings que está sendo criada.

Reviewer: ```suggestion
A flag `--name` define o nome da configuração de cache que está sendo criada.
```

### Flags opcionais

#### adaptive-delivery-action

> A opção `--adaptive-delivery-action` informa a ação de entrega adaptativa da configuração de Cache Settings. O valor padrão é `ignore`.

Reviewer: ```suggestion
A opção `--adaptive-delivery-action` informa a ativação da entrega adaptativa por device group da configuração de cache. O valor padrão é `ignore`.
```

#### browser-cache-settings

> A opção `--browser-cache-settings` configura o tempo que o conteúdo é armazenado em cache no navegador da web. O valor padrão é `honor`.

Reviewer: ```suggestion
A opção `--browser-cache-settings` configura o tempo que o conteúdo é armazenado em cache no navegador de usuários. O valor padrão é `honor`.
```

#### browser-cache-settings-maximum-ttl

A opção `--browser-cache-settings-maximum-ttl` informa o TTL máximo da configuração de Cache Settings do navegador.

#### cache-by-cookies

> A opção `--cache-by-cookies` define se o cache por cookies está ativo. O valor padrão é `ignore`.

Reviewer: ```suggestion
A opção `--cache-by-cookies` define se o conteúdo deve ser armazenado em cache de acordo com variações por cookies. O valor padrão é `ignore`.
```

Copiei do query string pq é a mesma coisa

#### cache-by-query-string

A opção `--cache-by-query-string` define como o conteúdo deve ser armazenado em cache de acordo com variações da query string em URLs. O valor padrão é `ignore`.

#### cdn-cache-settings

> A opção `--cdn-cache-settings` configura o tempo que as edge applications da Azion levam para armazenar em cache o conteúdo. Pode respeitar os cabeçalhos de cache de origem ou substituir as configurações de cache. O valor padrão é `honor`.

Reviewer: ```suggestion
A opção `--cdn-cache-settings` configura o tempo que as edge applications da Azion levam para armazenar o conteúdo em cache. Pode respeitar ou substituir os cabeçalhos de cache da origem. O valor padrão é `honor`.
```

#### cnd-cache-settings-maximum-ttl

> A opção `--cnd-cache-settings-maximum-ttl` informa o TTL máximo da configuração de Cache Settings da CDN. O valor padrão é `60`.

Reviewer: ```suggestion
A opção `--cnd-cache-settings-maximum-ttl` informa o TTL máximo de cache na edge. O valor padrão é `60`.
```

#### cookie-names

> A opção `--cookie-names` distingue objetos no cache da Azion pelo nome/valor dos cookies.

Reviewer: Essa tá relacionada com o cache-by-cookies, ou seja, precisa ser o outro valor diferente de ignore.

#### enable-caching-for-options

> A opção `--enable-caching-for-options` define se o cache para opções está ativo. O valor padrão é `false`.

Reviewer: ```suggestion
A opção `--enable-caching-for-options` define se o cache de método `OPTIONS` está ativo. O valor padrão é `false`.
```

#### enable-caching-for-post

A opção `--enable-caching-for-post` define se o cache para post está ativo.

#### enable-caching-string-sort

> A opção `--enable-caching-string-sort` define se a ordenação de strings em cache está ativa.

Reviewer: ```suggestion
A opção `--enable-caching-string-sort` define se a ordenação de query strings determina a diferença de objetos em cache.
```

Também depende do `--cache-by-query-string` não ser ignore.

#### file

> A opção `--file` especifica o caminho para um arquivo JSON contendo os atributos da configuração de Cache Settings que será criada. Você pode usar `-` para ler a partir do stdin.

Reviewer: ```suggestion
A opção `--file` especifica o caminho para um arquivo JSON contendo os atributos da configuração de cache que será criada. Você pode usar `-` para ler a partir do stdin.
```

#### help

A opção `--help` exibe mais informações sobre o comando `azion create cache-setting`.

#### l2-caching-enabled

A opção `--l2-caching-enabled` define se o L2 Caching está ativo.

#### query-string-fields

A opção `--query-string-fields` fornece uma lista de parâmetros de query string a serem considerados na configuração de Cache Settings, segregando o cache para a mesma URL.

#### slice-configuration-enabled

> A opção `--slice-configuration-enabled` define se a configuração de slice está ativa.

Reviewer: ```suggestion
A opção `--slice-configuration-enabled` define se a configuração de slice de arquivos grandes está ativa.
```

#### slice-configuration-range

> A opção `--slice-configuration-range` informa a faixa de configuração de slice.

Reviewer: ```suggestion
A opção `--slice-configuration-range` informa o tamanho de cada slice do arquivo.
```

#### sort

A opção `--sort` define a ordem dos itens na lista. O valor deve ser `asc` ou `desc`.

---

## Descrever

### Uso

```bash
$ azion describe cache-setting --application-id 1673635839 --cache-setting-id 107313
```

### Flags obrigatórias

#### application-id

A flag `--application-id` define o identificador único para a edge application.

#### cache-setting-id

A flag `--cache-setting-id` define o identificador único para uma configuração de Cache Settings.

### Flags opcionais

#### format

A opção `--format` altera o formato de saída. Passe o valor `json` para a opção para obter a saída em formato JSON.

#### help

A opção `--help` exibe mais informações sobre o comando `azion describe cache-setting`.

#### out

A opção `--out` exporta a saída para o caminho ou nome do arquivo fornecido com a extensão. Exemplo: `--out "./tmp/test.json"`.

---

## Atualizar

### Uso

```bash
$ azion update cache-setting --application-id 1673635839 --cache-setting-id 123123421 --name "phototypesetting"
```

### Flags obrigatórias

#### application-id

A flag `--application-id` define o identificador único para a edge application.

#### cache-setting-id

> A flag `--cache-setting-id` define o identificador único para uma configuração de Cache Settings.

Reviewer: ```suggestion
A flag `--cache-setting-id` define o identificador único para uma configuração de cache.
```

#### name

> A flag `--name` define o nome da configuração de Cache Settings que está sendo atualizada.

Reviewer: ```suggestion
A flag `--name` define o nome da configuração de cache que está sendo atualizada.
```

### Flags opcionais

#### adaptive-delivery-action

> A opção `--adaptive-delivery-action` informa a ação de entrega adaptativa da configuração de Cache Settings. O valor padrão é `ignore`.

Reviewer: ```suggestion
A opção `--adaptive-delivery-action` informa a ativação da entrega adaptativa por device group da configuração de cache. O valor padrão é `ignore`.
```

#### browser-cache-settings

> A opção `--browser-cache-settings` configura o tempo que o conteúdo é armazenado em cache no navegador da web. O valor padrão é `honor`.

Reviewer: ```suggestion
A opção `--browser-cache-settings` configura o tempo que o conteúdo é armazenado em cache no navegador de usuários. O valor padrão é `honor`.
```

#### browser-cache-settings-maximum-ttl

> A opção `--browser-cache-settings-maximum-ttl` informa o TTL máximo da configuração de Cache Settings do navegador.

Reviewer: ```suggestion
A opção `--browser-cache-settings-maximum-ttl` informa o TTL máximo do cache no navegador..
```

#### cache-by-cookies

> A opção `--cache-by-cookies` define se o cache por cookies está ativo. O valor padrão é `ignore`.

Reviewer: ```suggestion
A opção `--cache-by-cookies` define se o conteúdo deve ser armazenado em cache de acordo com variações por cookies. O valor padrão é `ignore`.
```

#### cache-by-query-string

> A opção `--cache-by-query-string` define como o conteúdo deve ser armazenado em cache de acordo com variações da query string em URLs. O valor padrão é `ignore`.

Reviewer: ```suggestion
A opção `--cache-by-query-string` define se a ordenação de query strings determina a diferença de objetos em cache. O valor padrão é `ignore`.
```

Faltou o valor padrão na outra vez que aparece o comando.

#### cdn-cache-settings

> A opção `--cdn-cache-settings` configura o tempo que as edge applications da Azion levam para armazenar em cache o conteúdo. Pode respeitar os cabeçalhos de cache de origem ou substituir as configurações de cache. O valor padrão é `honor`.

Reviewer: ```suggestion
A opção `--cdn-cache-settings` configura o tempo que as edge applications da Azion armazenam seu conteúdo em cache. Pode respeitar ou substituir os cabeçalhos de cache da origem. O valor padrão é `honor`.
```

#### cnd-cache-settings-maximum-ttl

> A opção `--cnd-cache-settings-maximum-ttl` informa o TTL máximo da configuração de Cache Settings da CDN. O valor padrão é `60`.

Reviewer: ```suggestion
A opção `--cnd-cache-settings-maximum-ttl` informa o TTL máximo do cache na edge. O valor padrão é `60`.
```

#### cookie-names

A opção `--cookie-names` distingue objetos no cache da Azion pelo nome/valor dos cookies.

#### enable-caching-for-options

> A opção `--enable-caching-for-options` define se o cache para opções está ativo. O valor padrão é `false`.

Reviewer: ```suggestion
A opção `--enable-caching-for-options` define se o cache para o método `OPTIONS` está ativo. O valor padrão é `false`.
```

#### enable-caching-for-post

A opção `--enable-caching-for-post` define se o cache para post está ativo.

#### enable-caching-string-sort

> A opção `--enable-caching-string-sort` define se a ordenação de strings em cache está ativa.

Reviewer: ```suggestion
A opção `--enable-caching-string-sort` define se a ordenação de query strings determina a diferença de objetos em cache.
```

#### file

> A opção `--file` especifica o caminho para um arquivo JSON contendo os atributos da configuração de Cache Settings que será atualizada. Você pode usar `-` para ler a partir do stdin.

Reviewer: ```suggestion
A opção `--file` especifica o caminho para um arquivo JSON contendo os atributos da configuração de cache que será atualizada. Você pode usar `-` para ler a partir do stdin.
```

#### help

A opção `--help` exibe mais informações sobre o comando `azion update cache-setting`.

#### l2-caching-enabled

A opção `--l2-caching-enabled` define se o L2 Caching está ativo.

#### query-string-fields

A opção `--query-string-fields` fornece uma lista de parâmetros de query string a serem considerados na configuração de Cache Settings, segregando o cache para a mesma URL.

#### slice-configuration-enabled

> A opção `--slice-configuration-enabled` define se a configuração de slice está ativa.

Reviewer: ```suggestion
A opção `--slice-configuration-enabled` define se a configuração de slice de arquivos grandes está ativa.
```

#### slice-configuration-range

> A opção `--slice-configuration-range` informa a faixa de configuração de slice.

Reviewer: ```suggestion
A opção `--slice-configuration-range` informa o tamanho de cada slice do arquivo.
```

#### slice-l2-caching-enabled

A opção `--slice-l2-caching-enabled` define se o slice do L2 Caching está ativo.

---

## Excluir

### Uso

```bash
$ azion delete cache-setting --application-id 1673635839 --cache-settings-id 107313
```

### Flags obrigatórias

#### application-id

A flag `--application-id` define o identificador único para a edge application.

#### cache-settings-id

> A flag `--cache-settings-id` define o identificador único para uma configuração de Cache Settings.

Reviewer: ```suggestion
A flag `--cache-settings-id` define o identificador único para uma configuração de cache.
```

---
title: Cache settings

> description: Crie, liste, descreva, atualize, e exclua suas configurações de cache na plataforma da Azion através da Azion CLI.

Reviewer 2: ```suggestion
description: Crie, liste, descreva, atualize e exclua suas configurações de cache na plataforma da Azion através da Azion CLI.
```

#### cnd-cache-settings-maximum-ttl

A opção `--cnd-cache-settings-maximum-ttl` informa o TTL máximo da configuração de Cache Settings da CDN. O valor padrão é `60`.

#### cookie-names

A opção `--cookie-names` distingue objetos no cache da Azion pelo nome/valor dos cookies.

#### enable-caching-for-options

A opção `--enable-caching-for-options` define se o cache para opções está ativo. O valor padrão é `false`.

#### enable-caching-for-post

A opção `--enable-caching-for-post` define se o cache para post está ativo.

#### enable-caching-string-sort

A opção `--enable-caching-string-sort` define se a ordenação de strings em cache está ativa.

#### file

A opção `--file` especifica o caminho para um arquivo JSON contendo os atributos da configuração de Cache Settings que será criada. Você pode usar `-` para ler a partir do stdin.

#### help

A opção `--help` exibe mais informações sobre o comando `azion create cache-setting`.

#### l2-caching-enabled

> A opção `--l2-caching-enabled` define se o cache L2 está ativo.

Reviewer 2: ```suggestion
A opção `--l2-caching-enabled` define se o L2 Caching está ativo.
```

#### query-string-fields

A opção `--query-string-fields` fornece uma lista de parâmetros de query string a serem considerados na configuração de Cache Settings, segregando o cache para a mesma URL.

#### slice-configuration-enabled

A opção `--slice-configuration-enabled` define se a configuração de slice está ativa.

#### slice-configuration-range

A opção `--slice-configuration-range` informa a faixa de configuração de slice.

#### slice-l2-caching-enabled

> A opção `--slice-l2-caching-enabled` define se o cache L2 de slice está ativo.

Reviewer 2: ```suggestion
A opção `--slice-l2-caching-enabled` define se o slice do L2 Caching está ativo.
```

---

## Atualizar

### Uso

```bash
$ azion update cache-setting --application-id 1673635839 --cache-setting-id 123123421 --name "phototypesetting"
```

### Flags obrigatórias

#### application-id

A flag `--application-id` define o identificador único para a edge application.

#### cache-setting-id

A flag `--cache-setting-id` define o identificador único para uma configuração de Cache Settings.

#### name

A flag `--name` define o nome da configuração de Cache Settings que está sendo atualizada.

### Flags opcionais

#### adaptive-delivery-action

A opção `--adaptive-delivery-action` informa a ação de entrega adaptativa da configuração de Cache Settings. O valor padrão é `ignore`.

#### browser-cache-settings

A opção `--browser-cache-settings` configura o tempo que o conteúdo é armazenado em cache no navegador da web. O valor padrão é `honor`.

#### browser-cache-settings-maximum-ttl

A opção `--browser-cache-settings-maximum-ttl` informa o TTL máximo da configuração de Cache Settings do navegador.

#### cache-by-cookies

A opção `--cache-by-cookies` define se o cache por cookies está ativo. O valor padrão é `ignore`.

#### cache-by-query-string

> A opção `--cache-by-query-string` define como o conteúdo deve ser armazenado em cache de acordo com var
> 
> iações da query string em URLs. O valor padrão é `ignore`.

Reviewer 2: ```suggestion
A opção `--cache-by-query-string` define como o conteúdo deve ser armazenado em cache de acordo com variações da query string em URLs. O valor padrão é `ignore`.
```

#### cnd-cache-settings-maximum-ttl

A opção `--cnd-cache-settings-maximum-ttl` informa o TTL máximo da configuração de Cache Settings da CDN. O valor padrão é `60`.

#### cookie-names

A opção `--cookie-names` distingue objetos no cache da Azion pelo nome/valor dos cookies.

#### enable-caching-for-options

A opção `--enable-caching-for-options` define se o cache para opções está ativo. O valor padrão é `false`.

#### enable-caching-for-post

A opção `--enable-caching-for-post` define se o cache para post está ativo.

#### enable-caching-string-sort

A opção `--enable-caching-string-sort` define se a ordenação de strings em cache está ativa.

#### file

A opção `--file` especifica o caminho para um arquivo JSON contendo os atributos da configuração de Cache Settings que será atualizada. Você pode usar `-` para ler a partir do stdin.

#### help

A opção `--help` exibe mais informações sobre o comando `azion update cache-setting`.

#### l2-caching-enabled

> A opção `--l2-caching-enabled` define se o cache L2 está ativo.

Reviewer 2: ```suggestion
A opção `--l2-caching-enabled` define se o L2 Caching está ativo.
```

#### query-string-fields

A opção `--query-string-fields` fornece uma lista de parâmetros de query string a serem considerados na configuração de Cache Settings, segregando o cache para a mesma URL.

#### slice-configuration-enabled

A opção `--slice-configuration-enabled` define se a configuração de slice está ativa.

#### slice-configuration-range

A opção `--slice-configuration-range` informa a faixa de configuração de slice.

#### slice-l2-caching-enabled

> A opção `--slice-l2-caching-enabled` define se o cache L2 de slice está ativo.

Reviewer 2: ```suggestion
A opção `--slice-l2-caching-enabled` define se o slice do L2 Caching está ativo.
```
